dashboard/assets/common/common.css

```diff
@@ -1,18 +1,3 @@
-#content {
```

Comment:
why is this moved to `main.css`? i'm guessing this is the better place for it. `main.css` is only the style for js front end, which is a sub section of the `#content` section.